service-fabric-secure-cluster-5-node-1-nodetype-wad/azuredeploy.json

```diff
@@ -0,0 +1,761 @@
+{
+  "$schema": "http://schema.management.azure.com/schemas/2014-04-01-preview/deploymentTemplate.json",
+  "contentVersion": "1.0.0.0",
+  "parameters": {
+    "clusterLocation": {
+      "type": "string",
+      "metadata": {
+        "description": "Location of the Cluster"
+      }
+    },
+    "clusterName": {
```

Comment:
@ChackDan But it could be the difference between the user entering the value "testcluster" or "uniqueString>ClusterName". The first will lead to deployment failures and user will need to start again.